docs/source/tutorials/multiple_pilots.ipynb

```diff
@@ -1,20 +1,420 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        td.arguments  = ['pilot_id=$RP_PILOT_ID']
</pre><p>Maybe adding a comment about what is the RP_PILOT_ID  and where can we find details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
@mturilli @andre-merzky should we provide a list of `RP_*` env variables available within the execution environment? should it be part of task description or configuration tutorial?